code/modules/reagents/chemistry/reagents.dm

```diff
@@ -33,6 +33,9 @@
 	. = ..()
 	holder = null
 
+/datum/reagent/proc/reaction_temperature(exposed_temperature, exposed_volume) //By default we do nothing.
```

Comment:
This is a new proc that basically can allow a temperature to do something when it's exposed to a certain temperature.

You may be thinking "but, are there chemical reactions that do this?"

Yes, that's very true, but some things are better reserved for this; for instance, maybe you want welding fuel to ignite at a certain temperature and spew flames, or maybe if it's an even hotter temperature, it just plain explodes.

This can't really be done with chemical recipes. Recipes are better reserved for simplistic once of reactions.